cmd/admin-handlers.go

```diff
@@ -294,6 +296,24 @@ type ServerDrivesPerfInfo struct {
 	Perf  []disk.Performance `json:"perf"`
 }
 
+// ServerCPUPerfInfo holds informantion about cpu utilization
+// of one minio node. It also reports any errors if encountered
+// while trying to reach this server.
+type ServerCPUPerfInfo struct {
+	Addr  string            `json:"addr"`
+	Error string            `json:"error,omitempty"`
+	Perf  []cpu.Performance `json:"perf"`
```

Comment:
> IMO we should have minioCPUUtil and totalCPUUtil fields. Total CPU utilization will be useful to troubleshoot cases where there is some other process overloading the CPU, while Minio CPU usage is normal..

I don’t think this makes sense. TotalCPU util has little to do with minio, and fluctuates wildly in quick succession. A snapshot in time CPU measure does not provide any value because the next clock tick could change the equation entirely. Even if we used the CPU metrics averaged over the last 1/5/15 minutes, it does not report the current utilization/current load imposed by minio. For instance, a total CPU utilization of 100% does not mean that minio is compute constrained - the metric is useless. However, a CPU utilization of 100% by minio would mean that it is compute constrained.

In order to infer a CPU bottleneck in a node where minio is not the cause, the utilization of minio process would suffice. I.e. minio process would report very high utilization. The clock ticks are only measured when the process is scheduled.

I think we should change this metric to CPU load, not util.